core/src/main/java/fj/data/NonEmptyList.java

```diff
@@ -395,13 +378,540 @@ public final A maximum(final Ord<A> o) {
    * @param obj the other object to check for equality against.
    * @return true if this list is equal to the provided argument
    */
-  @Override public boolean equals( final Object obj ) {
+  @Override
+  public boolean equals(final Object obj) {
     return Equal.equals0(NonEmptyList.class, this, obj, () -> Equal.nonEmptyListEqual(Equal.anyEqual()));
   }
 
-  @Override public int hashCode() {
+  @Override
+  public int hashCode() {
     return Hash.nonEmptyListHash(Hash.<A>anyHash()).hash(this);
   }
 
-  @Override public String toString() { return Show.nonEmptyListShow(Show.<A>anyShow()).showS(this); }
+  @Override
+  public String toString() {
+    return Show.nonEmptyListShow(Show.<A>anyShow()).showS(this);
+  }
+
+  /**
+   * Fold the list, from right to left.
+   *
+   * @param combine the combine function
+   * @return the output
+   */
+  public final A foldRight1(
```

Comment:
Prefer params on the same line as the method name unless it gets overly long, say >80 chars.